### DISCUSSION TITLE
希望添加 小度智能音响、小米、天猫精灵等 无线音响的支持

### DISCUSSION THREAD

**Discussion by JSZMonkey** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#92] [Ideas]
希望添加 小度智能音响、小米、天猫精灵等 无线音响的支持

**Comment by 1715173329** [comment]
如果它们是调用的网易云音乐接口，或许有希望。
btw I don't have these devices, so I don't know how to implement that.
